contracts/Tokens/VTokens/VToken.sol

```diff
@@ -522,15 +593,66 @@ contract VToken is VTokenInterface, Exponential, TokenErrorReporter {
         return result;
     }
 
+    function _accrueStableInterest(uint256 blockDelta) internal returns (uint256) {
```

Comment:
Comment needed

Reply:
Fixed in [commit](https://github.com/VenusProtocol/venus-protocol/pull/210/commits/6b9a0d6e81b2c8e15489d629723beb649c386738)